### PR TITLE
Handle csvcubed_uri_style transform setting for csvcubedintegration config loader

### PR DESCRIPTION
**Pull request by goofballLogic**

### CODE REVIEW

**Comment by goofballLogic** [comment]
Closes #356

File: gssutils/csvcubedintegration/configloaders/infojson.py [hunk 1]
```diff
@@ -93,8 +93,14 @@ def _from_info_json_dict(
...
         transform_section.get("columns", []), data, info_json_parent_dir
     )
+    uri_style = _uri_style_from_transform(transform_section)
 
-    return Cube(metadata, data, columns)
+    return Cube(metadata, data, columns, uri_style)
+
+def _uri_style_from_transform(transform_section):
+    if "csvcubed_uri_style" in transform_section:
+        return URIStyle[transform_section["csvcubed_uri_style"]]
```
Comment (goofballLogic) [inline gssutils/csvcubedintegration/configloaders/infojson.py:102]: do we need to catch enum parse errors here? Or just, Let It Fail:tm: ?

Comment (canwaf) [reply gssutils/csvcubedintegration/configloaders/infojson.py:102]: Just let it fail.

**Review by canwaf — APPROVED** [review]
Looks good. Ready for merging.

**Pull request merged**